packages/appstore/src/components/empty-state/__tests__/empty-state.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import EmptyState from '../empty-state';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
```

Comment:
@thisyahlen-deriv Why you need `@deriv/stores` for this test? 🤔

Reply:
oops 😸